### PR TITLE
minor fixes to README 

### PR DESCRIPTION
**Pull request by garymm**
Biggest changes:
* Fix references algorithm-specific sub-dirs (e.g., deep)
* Replace "exponential integration" with "exponential smoothing" in a few places. This term is much easier to find on e.g., Wikipedia.

### CODE REVIEW

**Comment by garymm** [comment]
Bump

File: README.md [hunk 1]
```diff
@@ -227,23 +228,24 @@ Metaphorically, various forms of equalizing taxation and wealth redistribution a
...
 
 3. **Zero-sum weight changes:** In some cases it can also be useful to constrain the faster error-driven weight changes to be zero-sum, which is supported by an optional parameter.  This zero-sum logic was nicely articulated by [Schraudolph (1998)](#references), and is implemented in the widely-used ResNet models.
 
 4. **Soft bounding and contrast enhancement:** To keep individual weight magnitudes bounded, we use a standard exponential-approach "soft bounding" dynamic (increases are multiplied by $1-w$; decreases by $w$).  In addition, as developed in the Leabra model, it is useful to add a *contrast enhancement* mechanism to counteract the compressive effects of this soft bounding, so that effective weights span the full range of weight values.
+TODO: what is $w$?
```
Comment (rcoreilly) [inline README.md:240]: w= weight -- specifically LWt -- could specify that more precisely but the generic form might be more recognizable to a broader audience -- so I wanted to leave this generic and it is then specified in the detailed eqs below.

File: README.md [hunk 2]
```diff
@@ -253,85 +255,85 @@ Axon is organized around a 200 msec *theta* cycle (5 Hz), which is perhaps not c
...
-* `ActM` = ActInt activation state at end of third quarter, representing the posterior-cortical minus phase activation -- used for statistics and monitoring network performance. Should not be used for learning or other computations.
-* `ActP` = ActInt activation state at end of fourth quarter, representing the posterior-cortical plus_phase activation -- used for statistics and monitoring network performance.  Should not be used for learning or other computations.
+* `Act` = rate-coded activation value reflecting instantaneous estimated rate of spiking, based on `1 / ISIAvg`.  This drives feedback inhibition in the FFFB function (todo: this will change when better inhibition is implemented), and is integrated over time for `ActInt` which is then used for performance statistics and layer average activations, etc.  Should not be used for learning or other computations.
```
Comment (rcoreilly) [inline README.md:262]: note: this has now happened and I will need to update.

File: README.md [hunk 3]
```diff
@@ -253,85 +255,85 @@ Axon is organized around a 200 msec *theta* cycle (5 Hz), which is perhaps not c
...
-* `AvgPct`= ActAvg as a proportion of overall layer activation -- this is used for synaptic scaling to match TrgAvg activation -- updated at SlowInterval intervals
-* `TrgAvg` = neuron's target average activation as a proportion of overall layer activation, assigned during weight initialization, driving synaptic scaling relative to AvgPct.
+* `Target` = target value: drives learning to produce this activation value in a supervised learning task.
+* `GeSyn` = time-integrated total excitatory synaptic conductance, with an instantaneous rise time from each spike (in `GeRaw`) and exponential decay with `Dt.GeTau`, aggregated over projections -- does *not* include `Gbar.E`.
+* `Ge` = total excitatory conductance, including all forms of excitation (e.g., NMDA) -- does *not* include `Gbar.E`.
+* `GiSyn` = time-integrated total inhibitory synaptic conductance, with an instantaneous rise time from each spike (in `GiRaw`) and exponential decay with `Dt.GiTau`, aggregated over projections -- does *not* include `Gbar.I`.  This is added with computed FFFB inhibition to get the full inhibition in `Gi`. TODO: I thought you said there are no inhibitory neurons, so I don't see how this fits in.
```
Comment (rcoreilly) [inline README.md:270]: we use the FS-FFFB function to approximate inhib neurons in most generic cortical models, but can still include actual inhib neurons, which are useful in various situations, especially in the BG.

File: README.md [hunk 4]
```diff
@@ -46,11 +46,11 @@ make -C axon release
 
 # Design and Organization
 
-* `ActParams` (in [act.go](https://github.com/emer/axon/blob/master/axon/act.go)), `InhibParams` (in [inhib.go](https://github.com/emer/axon/blob/master/axon/inhib.go)), and `LearnNeurParams` / `LearnSynParams` (in [learn.go](https://github.com/emer/axon/blob/master/axon/learn.go)) provide the core parameters and functions used.
+* `ActParams` (in [act.go](axon/act.go)), `InhibParams` (in [inhib.go](axon/inhib.go)), and `LearnNeurParams` / `LearnSynParams` (in [learn.go](axon/learn.go)) provide the core parameters and functions used.
```
Comment (rcoreilly) [inline README.md:49]: I thought for sure I tried this simple url path before and it failed -- great to know it works!

**Review by rcoreilly — APPROVED** [review]
All looks good -- hopefully we can merge once you update.  I will need to update again soon with recent updates.  

One thing I'm less sure about: the term "smoothing" seems conceptually different than "temporal integration" but that might just be my own idiosyncrasy -- maybe we can discuss tomorrow?

File: README.md (hunk 3 above)
Comment (garymm) [reply README.md:270]: Added a sentence about this to the `Inhibitory Competition Function Simulating Effects of Interneurons` section. PTAL and LMK if that's correct.

**Comment by garymm** [comment]
Regarding smoothing: I'm just using the term that seems to be the most common from my search on Wikipedia and Google that precisely describes the algorithm being used here.
It seems "exponentially weighted moving average" is also somewhat commonly used, and we can shorten it to EWMA.
"temporal integration" seems to be a much less specific term and therefore less useful. If you're trying to not be specific in some places, then maybe just "moving average" in those places, and "exponentially weighted moving average" or "exponential smothing" in the places where the details matter, like where you're talking about the constants used.

File: README.md [hunk 5]
```diff
@@ -106,29 +107,31 @@ Meanwhile, based on extensive experience with Axon and Leabra, here are some lik
...
 
-Thus, we are now using the [FS-FFFB](https://github.com/emer/axon/tree/master/fsfffb) *fast & slow* FFFB function that more explicitly captures the contributions of the PV+ and SST+ interneurons, and is based directly on FF and FB spikes, without requiring access to the internal Ge and Act rate-code variables in each neuron.  See above link for more info.  This function works even better overall than the original FFFB, in addition to providing a much more direct mapping onto the underlying biology.
+Thus, we are now using the [FS-FFFB](fsfffb) *fast & slow* FFFB function that more explicitly captures the contributions of the PV+ and SST+ interneurons, and is based directly on FF and FB spikes, without requiring access to the internal Ge and Act rate-code variables in each neuron.  See above link for more info.  This function works even better overall than the original FFFB, in addition to providing a much more direct mapping onto the underlying biology.
+
+While most layers and models use the FS-FFFB approximation, Axon does support explicit modelling of inhibitory neurons by using the `emer.Inhib` value of `emer.PrjnType`.
```
Comment (rcoreilly) [inline README.md:134]: yep!

**Comment by garymm** [comment]
Switched to "exponentially weighted moving average" and "EWMA"

**Review by rcoreilly — APPROVED** [review]
another acronym to master :)  Anyway, good to be accurate!

**Pull request merged**